shared/xccdf/system/auditing.xml

```diff
@@ -896,11 +896,13 @@ cui="3.3.1" />
 
 <Rule id="file_ownership_var_log_audit" severity="medium" prodtype="rhel7">
 <title>System Audit Logs Must Be Owned By Root</title>
-<description>
-<fileowner-desc-macro file="/var/log" owner="root"/>
+<description>All Audit logs must be owned by root user and group, by default the path for Audit log entries is /var/log/audit/.
```

Comment:
@rsprudencio please make the following update:
`All audit logs must be owned by root user and group. By default, the path for audit log is <pre>/var/log/audit/</pre>.